<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Video">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>551</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid #2e2e2e;
    border-radius: 10px;
    background-color:  rgba(30, 30, 30, 180);
    margin-top: 0px;
}

</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_retroceder">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8e8481;
    border-radius: 35px;  /* Mitad del tamaño del ancho y alto para hacer un círculo */
    background-color: #938f8e;
    color: white;
    width: 50px;  /* Anchura del botón */
    height: 50px; /* Altura del botón */
}

QPushButton:pressed {
    background-color: #3e3e3e;  /* Color cuando se presiona */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>41</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="progressBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pausar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8e8481;
    border-radius: 35px;  /* Mitad del tamaño del ancho y alto para hacer un círculo */
    background-color: #938f8e;
    color: white;
    width: 50px;  /* Anchura del botón */
    height: 50px; /* Altura del botón */
}

QPushButton:pressed {
    background-color: #3e3e3e;  /* Color cuando se presiona */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_adelantar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8e8481;
    border-radius: 35px;  /* Mitad del tamaño del ancho y alto para hacer un círculo */
    background-color: #938f8e;
    color: white;
    width: 50px;  /* Anchura del botón */
    height: 50px; /* Altura del botón */
}

QPushButton:pressed {
    background-color: #3e3e3e;  /* Color cuando se presiona */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8e8481;
    border-radius: 35px;  /* Mitad del tamaño del ancho y alto para hacer un círculo */
    background-color: #938f8e;
    color: white;
    width: 50px;  /* Anchura del botón */
    height: 50px; /* Altura del botón */
}

QPushButton:pressed {
    background-color: #3e3e3e;  /* Color cuando se presiona */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8e8481;
    border-radius: 35px;  /* Mitad del tamaño del ancho y alto para hacer un círculo */
    background-color: #938f8e;
    color: white;
    width: 50px;  /* Anchura del botón */
    height: 50px; /* Altura del botón */
}

QPushButton:pressed {
    background-color: #3e3e3e;  /* Color cuando se presiona */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mute">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8e8481;
    border-radius: 35px;  /* Mitad del tamaño del ancho y alto para hacer un círculo */
    background-color: #938f8e;
    color: white;
    width: 50px;  /* Anchura del botón */
    height: 50px; /* Altura del botón */
}

QPushButton:pressed {
    background-color: #3e3e3e;  /* Color cuando se presiona */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8e8481;
    border-radius: 35px;  /* Mitad del tamaño del ancho y alto para hacer un círculo */
    background-color: #938f8e;
    color: white;
    width: 50px;  /* Anchura del botón */
    height: 50px; /* Altura del botón */
}

QPushButton:pressed {
    background-color: #3e3e3e;  /* Color cuando se presiona */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>211</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
